pom.xml

```diff
@@ -2357,6 +2363,17 @@
                     </configuration>
                 </plugin>
 
+                <plugin>
+                    <groupId>org.basepom.maven</groupId>
+                    <artifactId>duplicate-finder-maven-plugin</artifactId>
+                    <configuration>
+                        <ignoredClassPatterns combine.children="append">
```

Comment:
Why do we need `combine.children="append"` here?

Reply:
Instead of adding this plugin to several child modules, we can add this in the root and this will cause it to be inherited.